src/docs/asciidoc/web.adoc

```diff
@@ -1,25 +1,30 @@
 [[spring-web]]
-= Web
+= The Web
 :doc-root: https://docs.spring.io
 :api-spring-framework: {doc-root}/spring-framework/docs/{spring-version}/javadoc-api/org/springframework
 :toc: left
 :toclevels: 2
 
-This part of the documentation covers support for web applications designed to run on a
-traditional Servlet stack (Servlet API + Servlet container).
+This part of the reference documentation covers Spring Framework's support for the
+presentation tier (and specifically web-based presentation tiers) including support
+for WebSocket-style messaging in web applications.
 
-Chapters cover the Servlet-based <<mvc,Spring MVC>> web framework including <<mvc-view,Views>>,
-<<mvc-cors,CORS>>, and <<websocket,WebSocket>> support.
+Spring Framework's own web framework, <<mvc,Spring Web MVC>>, is covered in the
+first couple of chapters. Subsequent chapters are concerned with Spring Framework's
+integration with other web technologies, such as <<jsf,JSF>>.
 
-Note that as of Spring Framework 5.0 web applications can also run on a
-<<spring-web-reactive, reactive web stack>> (Reactive Streams API + non-blocking runtime).
+The section then concludes with comprehensive coverage of the Spring Framework
```

Comment:
All links except "reactive web stack" are good, so these changes are not relevant, as stated previously web and reactive web documentation are now splitted.